rustConsole/.vscode/launch.json

```diff
@@ -12,7 +12,6 @@
             "environment": [],
             "externalConsole": false,
             "MIMode": "gdb",
-            "miDebuggerPath": "/usr/bin/rust-gdb",
```

Comment:
Because the path changes when installing Rust with `rustup`, so this was leading to issues, but if this is removed VSCode gets the default path for Rust, and then it works